packages/data/src/subgraph.ts

```diff
@@ -202,6 +202,26 @@ export default class SemaphoreSubgraph {
         return groups[0]
     }
 
+    /**
+     * Returns a list of group members.
+     * @param groupId Group id.
+     * @returns Group members.
+     */
+    async getGroupMembers(groupId: string): Promise<string[]> {
+        const group = await this.getGroup(groupId, { members: true }) // parameters are checked inside getGroup
+        return group.members ?? []
```

Comment:
`group.members` should be either an empty list or a list of members. So `?? []` may not be necessary. Have you checked it?

Reply:
Yes, it should work in the same way 👍🏽